### PR TITLE
[PREL-36] Add 11.8 and 12.3 ppg support

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: scripts/percona-release.sh [hunk 1]
```diff
@@ -299,14 +305,16 @@ function enable_alias {
...
   [[ ${NAME} = pdmdb4.2 ]] && REPOS=${PDMDB4_2_REPOS:-}
   [[ ${NAME} = pdmdb4.2.6 ]] && REPOS=${PDMDB4_2_6_REPOS:-}
   [[ ${NAME} = ppg12 ]] && REPOS=${PPG12_REPOS:-}
   [[ ${NAME} = ppg12.2 ]] && REPOS=${PPG12_2_REPOS:-}
+  [[ ${NAME} = ppg12.3 ]] && REPOS=${PPG12_3_REPOS:-}
   [[ ${NAME} = pdmysql8.0 ]] && REPOS=${PDMYSQL80_REPOS:-}
   [[ ${NAME} = pdmysql8.0.18 ]] && REPOS=${PDMYSQL80_18_REPOS:-}
   [[ -z ${REPOS} ]] && REPOS="original tools"
-  if [[ ${NAME} = ps80 ]] || [[ ${NAME} = pxc80 ]] || [[ ${NAME} = ppg11 ]] || [[ ${NAME} = ppg11.5 ]] || [[ ${NAME} = ppg11.6 ]] || [[ ${NAME} = ppg11.7 ]] || [[ ${NAME} = ppg12 ]] || [[ ${NAME} = ppg12.2 ]] || [[ ${NAME} = pdmysql8.0 ]] || [[ ${NAME} = pdmysql8.0.18 ]]; then
+  if [[ ${NAME} = ps80 ]] || [[ ${NAME} = pxc80 ]] || [[ ${NAME} = ppg11 ]] || [[ ${NAME} = ppg11.5 ]] || [[ ${NAME} = ppg11.6 ]] || [[ ${NAME} = ppg11.7 ]]  | [[ ${NAME} = ppg11.8 ]] || [[ ${NAME} = ppg12 ]] || [[ ${NAME} = ppg12.2 ]] || [[ ${NAME} = ppg12.3 ]] || [[ ${NAME} = pdmysql8.0 ]] || [[ ${NAME} = pdmysql8.0.18 ]]; then
```
Comment (vorsel) [inline scripts/percona-release.sh:317]: missing | in || operator

File: scripts/percona-release.sh [hunk 2]
```diff
@@ -9,9 +9,9 @@ if [[ $(id -u) -gt 0 ]]; then
   exit 1
 fi
 #
-ALIASES="ps56 ps57 ps80 psmdb34 psmdb36 psmdb40 psmdb42 pxb80 pxc56 pxc57 pxc80 ppg11 ppg11.5 ppg11.6 ppg11.7 ppg12 ppg12.2 pdmdb4.2 pdmdb4.2.6 pdmysql8.0.18 pdmysql8.0"
+ALIASES="ps56 ps57 ps80 psmdb34 psmdb36 psmdb40 psmdb42 pxb80 pxc56 pxc57 pxc80 ppg11 ppg11.5 ppg11.6 ppg11.7  ppg11.8 ppg12 ppg12.2 ppg12.3 pdmdb4.2 pdmdb4.2.6 pdmysql8.0.18 pdmysql8.0"
```
Comment (vorsel) [inline scripts/percona-release.sh:12]: extra space before ppg11.8

File: scripts/percona-release.sh (hunk 1 above)
Comment (EvgeniyPatlan) [reply scripts/percona-release.sh]: fixed

File: scripts/percona-release.sh (hunk 2 above)
Comment (EvgeniyPatlan) [reply scripts/percona-release.sh]: fixed

**Pull request merged**